sdk/communication/Azure.Communication.CallingServer/src/Models/Events/AddParticipantsFailed.cs

```diff
@@ -23,21 +23,29 @@ internal AddParticipantsFailed()
         /// <param name="internalEvent">Internal Representation of the AddParticipantsFailedEvent. </param>
         internal AddParticipantsFailed(AddParticipantsFailedInternal internalEvent)
         {
+            EventSource = internalEvent.EventSource;
             OperationContext = internalEvent.OperationContext;
-            ResultInfo = internalEvent.ResultInfo;
+            ResultInformation = internalEvent.ResultInformation;
             Participants = internalEvent.Participants.Select(t => CommunicationIdentifierSerializer.Deserialize(t)).ToList();
-            EventType = internalEvent.EventType;
+            Version = internalEvent.Version;
             CallConnectionId = internalEvent.CallConnectionId;
             ServerCallId = internalEvent.ServerCallId;
             CorrelationId = internalEvent.CorrelationId;
+            PublicEventType = internalEvent.PublicEventType;
         }
 
+        /// <summary> EventSource. </summary>
+        public string EventSource { get; }
```

Comment:
This is part of the CloudEvent envelope. The "Data" property is all we need to create properties for.